pkg/tsdb/azuremonitor/azuremonitor-datasource.go

```diff
@@ -223,15 +223,15 @@ func (e *AzureMonitorDatasource) createRequest(ctx context.Context, dsInfo *mode
 		return nil, errors.New("Unable to find datasource plugin Azure Monitor")
 	}
 
+	cloudName := dsInfo.JsonData.Get("cloudName").MustString("azuremonitor")
 	var azureMonitorRoute *plugins.AppPluginRoute
 	for _, route := range plugin.Routes {
-		if route.Path == "azuremonitor" {
+		if strings.Contains(route.Path, cloudName) {
```

Comment:
Shouldn't it still be equals here?

Reply:
Yes, I think somehow I confused myself when looking at the plugin json and where cloudName comes from in the context of the different services.